editor/editor_resource_picker.cpp

```diff
@@ -943,6 +926,7 @@ void EditorResourcePicker::_ensure_resource_menu() {
 	add_child(edit_menu);
 	edit_menu->connect("id_pressed", callable_mp(this, &EditorResourcePicker::_edit_menu_cbk));
 	edit_menu->connect("popup_hide", callable_mp((BaseButton *)edit_button, &BaseButton::set_pressed).bind(false));
+	edit_menu->add_theme_constant_override("icon_max_width", get_theme_constant(SNAME("class_icon_size"), EditorStringName(Editor)));
```

Comment:
We would generally put all configuration lines before `add_child` (except signal connects, which we usually put after `add_child`, like it's done here). But this isn't critical.

Reply:
Ah that is good to know! Fixed & I will keep this in mind for next time. Thank you!